platform/mime/core/platform-mime-core-impl/pom.xml

```diff
@@ -75,6 +77,15 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.codice.thirdparty</groupId>
+            <artifactId>tika-bundle</artifactId>
```

Comment:
All the other usages in the project also use it as a dependency in this way. Is that tech debt?